demos/android/MASVS-RESILIENCE/MASTG-DEMO-0038/MASTG-DEMO-0038.md

```diff
@@ -4,6 +4,34 @@ title: Detecting StrictMode Uses with Frida
 id: MASTG-DEMO-0038
 code: [kotlin]
 test: MASTG-TEST-0264
-status: draft
-note: This demo shows how to detect the use of StrictMode at runtime using Frida.
+status: new
 ---
+
+### Sample
+
+This sample demonstrates the detection of `StrictMode` uses at runtime using Frida. The app enables a `StrictMode` policy to detect leaked SQLite objects and intentionally leaves a cursor unclosed to trigger the policy.
+
+{{ ../MASTG-DEMO-0037/MastgTest.kt }}
+
+### Steps
+
+1. Install the app on a device (@MASTG-TECH-0005)
+2. Make sure you have @MASTG-TOOL-0001 installed on your machine and the frida-server running on the device
```

Comment:
```suggestion
2. Make sure you have @MASTG-TOOL-0031 installed on your machine and the frida-server running on the device
```

Reply:
Also: shouldn't this (eventually?) be a Technique?

Reply:
Maybe yes. We can discuss this on our next call. I'll resolve for now since 0001 is correct (Frida for Android)